node/cn/handler_test.go

```diff
@@ -112,6 +112,24 @@ func newBlock(blockNum int) *types.Block {
 	return block
 }
 
+func newBlockWithParentHash(blockNum int, parentHash common.Hash) *types.Block {
```

Comment:
Why is this function needed?

Reply:
In `TestPrivateDebugAPI_computeTxEnv` unit test, there is a logic to check the parent hash of the block.